docs/controls/access-controls/ac-2(3).md

```diff
@@ -0,0 +1,14 @@
+# Access Control
+## AC-02(03) - DISABLE INACTIVE ACCOUNTS  
+
+The information system automatically disables inactive accounts after [a maximum of 60 days of inactivity].  
```

Comment:
Our users only log in once per quarter, so this will be 180 days in TDP

Reply:
This may need to be discussed and possibly need an approved exception since this is a NIST / FedRAMP requirement.